src/examples/demo.ts

```diff
@@ -25,6 +25,8 @@ const props: WorkspaceProps & ClassAttributes<Workspace> = {
         window.location.hash = saveLayoutToLocalStorage(layoutData);
         window.location.reload();
     },
+    leftPanelInitiallyOpen: false,
```

Comment:
I would probable leave default values for this props in the example (to keep it simple).